epigraphhub/data/brasil/sinan/extract.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import PosixPath
+
+from loguru import logger
+from pysus.online_data import SINAN
+
+from epigraphhub.data._config import SINAN_LOG_PATH
+
+logger.add(SINAN_LOG_PATH, retention="7 days")
+
+aggravates = SINAN.agravos
+
+
+def download(aggravate: str):
```

Comment:
About this subject, I've changed after my wife points that "Violência Domestima" is not a disease. But I'll change it back

Reply:
Yes, not everything in SINAN is a disease, but the majority is. If we were to seek a more direct translation of  "Agravos à Saúde", which is the name in portuguese, the best alternative would be IMO "Health threats" But in the epidemiology, many threats such as "domestic violence" or "Poisonous animal bites" are treated as a disease because they really affect the overall health of the population. Even the [definition of disease](https://en.wikipedia.org/wiki/Disease), is not as simple as it seems! :-)